.github/workflows/deploy-latest-helm-version.yaml

```diff
@@ -19,6 +19,6 @@ jobs:
           gh workflow run updatecli-helm-integration-tests.yaml \
             --repo prefecthq/ops-cluster-deployment \
             --ref main \
-            -f helm_version=${{ github.ref_name }}
+            -f release_version=${{ github.release_version }}
```

Comment:
I think this is meant to be `inputs.release_version`

Reply:
good catch